Description: Switch to the Debian path for iso flags
Author: marga@debian.org
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
---
 src/passwordEntry.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- cinnamon-screensaver.git.orig/src/passwordEntry.py
+++ cinnamon-screensaver.git/src/passwordEntry.py
@@ -71,7 +71,7 @@ class PasswordEntry(Gtk.Entry):
             name = self.keyboard_controller.get_current_icon_name()
 
             if name:
-                filename = "/usr/share/iso-flag-png/%s.png" % name
+                filename = "/usr/share/iso-flags-png-320x240/%s.png" % name
 
                 try:
                     pixbuf = GdkPixbuf.Pixbuf.new_from_file_at_size(filename, -1, height)
